Detectors/TPC/simulation/include/TPCSimulation/DigitCRU.h

```diff
@@ -35,7 +35,7 @@ class DigitCRU{
     
     /// Constructor
     /// \param mCRU CRU ID
-    DigitCRU(int mCRU);
+    DigitCRU(int mCRU, const CommonModeContainer &commonModeCont);
```

Comment:
I think I said const ref. But if the object needs to be modified in DigitCRU I'm not sure this will work. Perhaps you need to remove the const ...